Geometry_Engine/Compute/CollapseToScaledPolyline.cs

```diff
@@ -61,17 +61,17 @@ public static Polyline CollapseToScaledPolyline(this PolyCurve curve, double tol
                 scale = Math.Max(totalHeight, totalWidth) / 2;
             }
 
-            double scaledTolerance = scale * toleranceGrowth;
+            double scaledTolerance = scale * toleranceScaleFactor;
```

Comment:
I didn't get why you are using two factors to control the refinement rate of the curves. Maybe it would be easier for the user to use just one of them?

Reply:
probably but that would just hide an hard-coded value inside the method no?

And it factors the tolerance to its scale, so `toleranceScaleFactor`